File: proxy/v1/message_interaction/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message_interaction = @client.proxy
  .services('KSaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .sessions('KCaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .participants('KPaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .message_interactions('KIaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .fetch

puts message_interaction.sid

File: trunking/v1/ip_access_control_list/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Trunking.V1.Trunk;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var ipAccessControlList = IpAccessControlListResource.Fetch(
            pathTrunkSid: "TRaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathSid: "ALaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(ipAccessControlList.AccountSid);
    }
}